internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
Happens all the time to me! I do like all of your ideas about Test-SqlConnection. I forgot, would Kerberos go in there? 

Yeah  I agree that this command would be complete and would also like the Test-Connection in the first one -- I find it to be fastest, but that may be my bias. You?